hook-tests/600-no-debian.test

```diff
@@ -0,0 +1,268 @@
+#!/bin/sh
+#
+# test for hook that creates dpkg.yaml used for OSS compliance
+
+#first check if file exists
+FILE=usr/share/snappy/dpkg.yaml
+test -e $FILE
+
+#second check the PPA section of dpkg.yaml
+ppa_section=$(cat <<EOF
+package-repositories:
+- type: apt
+  ppa: canonical-foundations/ubuntu/ubuntu-image
+EOF
+)
+
+if grep -Fxq "$ppa_section" $FILE
+then
+  echo "Content of PPA section is as expected in dpkg.yaml."
+else 
+  echo "Error! PPA section of dpkg.yaml not as expected."
+  echo "If that is intentional, please update the ppa_section content in the"
+  echo "test file: core18/hook-tests/602-create-dpkg-yaml.test."
+  exit 1
+fi
+
+pkgs_start=$(( $(awk '/packages:/{ print NR; exit }' $FILE) +1))
+in_yaml_file_pkgs=$(cat <<EOF> in_yaml_file_pkgs.txt
+$(tail -n +$pkgs_start $FILE | awk -F= '{print $1}' | awk -F "- " '{print $NF }'|sort)
+EOF
+)
+
+expected_pkgs=$(sort cat <<EOF> expected_pkgs.txt
```

Comment:
Right now this generates an error `sort: cannot read: cat: No such file or directory` and generates an empty `expected_pkgs.txt`.
Should this simply be `expected_pkgs=$(sort <<EOF> expected_pkgs.txt` instead?